flyway-community-db-support/pom.xml

```diff
@@ -45,12 +45,22 @@
             <version>${version.junit}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
         <!-- JDBC drivers -->
         <dependency>
             <groupId>org.apache.ignite</groupId>
             <artifactId>ignite-core</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>com.clickhouse</groupId>
+            <artifactId>clickhouse-jdbc</artifactId>
+            <version>0.4.0</version>
+            <optional>true</optional>
```

Comment:
Why dependency is optional and driver not included into build by default?

Reply:
Because you may want to use a different version of driver or even legacy driver from "ru.yandex" package.

Reply:
You can find that all drivers are optional in pom.xml for flyway-core and flyway-community-db-support modules. Your app has to include a proper driver.